nginx/Dockerfile

```diff
@@ -1,5 +1,21 @@
-FROM nginxinc/nginx-unprivileged:1.20
+FROM nginx:latest
+
 COPY nginx.conf /etc/nginx/nginx.conf
-COPY security.conf /etc/nginx/security.conf
-COPY default.nginx /etc/nginx/conf.d/default.conf
-USER 1000
\ No newline at end of file
+
+# configs
+COPY config/general.conf /etc/nginx/config/general.conf
```

Comment:
Can this be done in one copy command? Or is there files you dont want to override